src/screens/SendCollectible/SendCollectibleConfirm.js

```diff
@@ -88,19 +89,22 @@ const WarningMessage = styled(Paragraph)`
 const SendCollectibleConfirm = ({
   isOnline,
   balances,
-  navigation,
   resetEstimateTransaction,
   estimateTransaction,
   keyBasedWalletAddress,
   feeInfo,
   isEstimating,
   estimateErrorMessage,
 }: Props) => {
+  const navigation = useNavigation();
+
+  const receiverEnsName = useNavigationParam('receiverEnsName');
+  const assetData = useNavigationParam('assetData');
+  const receiver = useNavigationParam('receiver');
+  const navigationSource = useNavigationParam('source');
+  const chain = useNavigationParam('chain');
```

Comment:
Pls add explicit typing to these params, consider adding fallback code if any if the params is null/underifned

```suggestion
  const receiverEnsName = useNavigationParam('receiverEnsName');
  const assetData = useNavigationParam('assetData');
  const receiver = useNavigationParam('receiver');
  const navigationSource = useNavigationParam('source');
  const chain = useNavigationParam('chain');
```